analyzer/src/testFixtures/kotlin/Extensions.kt

```diff
@@ -97,3 +102,18 @@ private fun Project.filterReferencedPackages(allPackages: Set<Package>): Set<Pac
 fun ProjectAnalyzerResult.withInvariantIssues() =
     // Account for different NPM versions to return issues in different order.
     copy(issues = issues.sortedBy { it.message }.map { it.copy(timestamp = Instant.EPOCH) })
+
+fun Spec.analyze(
+    projectDir: File,
+    allowDynamicVersions: Boolean = false,
+    packageManagers: Collection<PackageManagerFactory> = PackageManager.ALL.values
+): OrtResult {
+    val config = AnalyzerConfiguration(
+        allowDynamicVersions = allowDynamicVersions
+    )
```

Comment:
Nit: It probably makes to simply write this as `val config = AnalyzerConfiguration(allowDynamicVersions)`.